dictsource/ru_rules

```diff
@@ -1,544 +1,566 @@
+        б (хY   p
+        б (хкY  p
+        б (ч    p
+        б (щ    p
+        б (ь_   p;
+
+.group в
+        в       v
+?2      в (_    v-
+        в (_    f
+        в (Y    v;
+        в (бY   v;
+        в (вY   v;
+        в (вбY  v;
+        в (вмY  v;
+        в (кY   f
+        в (мY   v;
+        в (п'   f;
+        в (сY   f
+        в (тY   f
+        в (ф'   f;
+        в (хY   f
+        в (хкY  f
+        в (ч    f
+        в (щ    f
+        в (ь_   f;
+        в (ьт   f;
+     здра) в (ств         // silent v(не произносимая в)
+    чу) в (ств      // silent v(не произносимая в)  
+
+.group г
+        г       g
+    Cе) г (о_   v
+    Cо) г (о_   v
+  _все) г (о_   v
+        г (Y    g;
+        г (_    k
+        г (к    x
+        г (пY   k
+        г (сY   k
+        г (тY   k
+        г (фY   k
+        г (ффY  k
+        г (хY   k
+        г (хкY  k
+        ле) г (ч    x
+        _доро) г (о   g
+    _е) г (о_   v
+     е) г (ося_ v
+        _ле) г (о_       g  // лего, конструктор lego, до исправления читал как лево
+    _ло) г (о_      g  // сокращенное от логотип, до исправления читал как лово.
+  _мно) г (о_   g
+  _мое) г (о_   v
+   _намно) г (о_   g
+   _ненамно) г (о_   g
+(некое) г (о            v
+     _убо) г (о_         g
+            _настро) г (о_           g
+            _нестро) г (о_           g
+   _не) г (о_   v
+   _недоро) г (о_   g   
+   _занедоро) г (о_   g   
+     _немно) г (о_   g
+       _примно) г (о_               g
+   _се) г (о    v
+             _страте) г (о_               g // stratego board game
+   _стро) г (о   g
+ _твое) г (о_   v
+ _свое) г (о_   v
+
+.group д
+        д       d
+        д (B    d;
+        д (Y    d;
+        д (_    t
+            д (вк          t
+        д (дY   d;
+        д (зY   d;
+        д (к   t
+        д (нь   d;
+        д (п   t
+        д (с   t
+        д (сY   t;
+        д (тY   t;
+        д (фY   t
+        д (ффY  t
+        д (х   t
+        д (хкY  t
+             д (ш             t
+         д (ч               t
+             д (ц             t
+        д (ь_   t;
+        д (ьк   t;
+         _бан) д (жо_             // silent д (непроизносимая д)
+      з) д (н        // silent d(Непроизносимая д)
+    р) д (ц     // Silent D(не произносимая д)
+          мо) де (м        dE%
+             н) де (кс     dE2 // яндекс, индекс/ yandex, index
+         _мо) де (л               dE
+                 _мо) де (лир              dE%
+               мо) де (льер              dE%
+            мо) де (лист              dE%
+          ше) де (вр               dE
+
+.group е
+        е          e
+?2      A) е (_    ;e
+        A) е       je
+        A) е (_    jI3     //second position
+        _) е       jI3     //second position
+        т) е (ст   E%
+        про) е (кт    E
+        ъ) е       je
+        ъ) е (_    jI3     //second position
+        ь) е       je
+        ь) е (_    jI3     //second position
+        _) е ((́   ''je   

```

Comment:
You probably need to remove later the extra opening parenthesis where there are two consecutive (. You can check the documentation in the link below.

https://github.com/espeak-ng/espeak-ng/blob/master/docs/dictionary.md#rules